cloudflare/resource_cloudflare_access_policy_test.go

```diff
@@ -129,3 +119,66 @@ func testAccessPolicyAnyServiceTokenConfig(resourceID, zone, zoneID string) stri
 
 	`, resourceID, zone, zoneID)
 }
+
+func TestAccessPolicyGroup(t *testing.T) {
```

Comment:
This is also failing

```
------- Stdout: -------
=== RUN   TestAccessPolicyGroup
--- FAIL: TestAccessPolicyGroup (3.40s)
    testing.go:654: Step 0 error: errors during apply:
        
        Error: error creating Access Policy for ID "": error from makeRequest: HTTP status 400: content "{\n  \"result\": null,\n  \"success\": false,\n  \"errors\": [\n    {\n      \"code\": 11017,\n      \"message\": \"access.api.error.invalid_policy\"\n    }\n  ],\n  \"messages\": []\n}\n"
        
          on /opt/teamcity-agent/temp/buildTmp/tf-test053014304/main.tf line 17:
          (source code not available)
        
        
FAIL
```

You can run these locally by following the instructions for [developing the provider from the README](https://github.com/terraform-providers/terraform-provider-cloudflare#developing-the-provider).

Reply:
It wasn't easy to figure out what's the problem cause message was not so obvious.

But after a while debugging I figured out, that it was complaining about trying to add Access Group not containing non-identity rules (here: email) to non identity policy.
So I changed in test `cloudflare_access_group`
```
include {
   email = ["test@example.com"]
}
```
to
```
include {
   ip = ["127.0.0.1"]
}
```

After that change now test passes 😀 

Thanks for help!